app/service.js

```diff
@@ -98,7 +98,9 @@ app.get('/__version__', function(req, res) {
 // The error handler must be before any other error middleware
 app.use(raven.middleware.express.errorHandler(sentryDSN));
 
-app.listen(7001, function() {
+const port = Number.isInteger(process.env.PORT) ? process.env.PORT : 7001;
```

Comment:
I think you can pass a port number as a string to Express and it will cast it appropriately.

```js
var express = require('express');
var app = express();
app.get('/', (req, res) => {
  res.send("OK!!");
});
const server = app.listen("3009", () => {
  console.log(`listening on ${server.address().port}`); // listening on 3009
});
```

You may be able to simplify this to:

```js
const port = process.env.PORT || 7001;
app.listen(port, function () {
});
```

---

... Actually, your code will fail if you pass a port via the CLI since Node/Bash seems to treat those as strings:

```sh
$ PORT=7003 node index
```
```js
var express = require('express');
var app = express();
app.get('/', (req, res) => {
  res.send("OK!!");
});
const port = Number.isInteger(process.env.PORT) ? process.env.PORT : 7001;
console.log('process.env.PORT:', process.env.PORT); // 7008
console.log('typeof process.env.PORT:', typeof process.env.PORT); // string
console.log('Number.isInteger(process.env.PORT):', Number.isInteger(process.env.PORT)); // false
console.log('port:', port); // 7001
console.log('typeof port:', typeof port); // number
const server = app.listen(port, () => {
});
```

So even though I'm explicitly passing a `PORT` via the CLI, Node.js thinks it's a string, your `Number.isInteger()` check fails, and it uses the default port 7001 anyways. :shrug: